### ISSUE TITLE
Many spaces in block comment result in having to run rustfmt twice

### ISSUE THREAD

**Issue by faern** [rust-lang-nursery/rustfmt#1203]
If I have a one line block comment with many trailing spaces in, rustfmt removes the trailing spaces in the first pass. However, the rest of that statement can sometimes be formatted in a way that will get further adjusted if rustfmt is executed a second time. Thus rustfmt does not always reach desired final format on the first run.

This has been tested on rustfmt 0.5.0 and 0.6.2 with the same result.

Original code:

``` rust
fn main() {
    let a = [1 /* A comment                                                    */, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17];
}
```

After running rustfmt one time:

``` rust
fn main() {
    let a = [1 /* A comment */, 2, 3, 4,
             5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17];
}
```

After running rustfmt a second time:

``` rust
fn main() {
    let a = [1 /* A comment */, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17];
}
```

**Comment by topecongiro** [comment]
Currently (0.8.3 ()) this is reproduced when `normalize_comments = true`.

**Comment by topecongiro** [comment]
Fixed on `0.2.16-nightly`, closing.
